runtime/src/lib.rs

```diff
@@ -70,12 +72,22 @@ pub use content_directory::{
     HashedTextMaxLength, InputValidationLengthConstraint, MaxNumber, TextMaxLength, VecMaxLength,
 };
 
+#[cfg(feature = "std")]
+/// Wasm binary unwrapped. If built with `BUILD_DUMMY_WASM_BINARY`, the function panics.
+pub fn wasm_binary_unwrap() -> &'static [u8] {
+    WASM_BINARY.expect(
+        "Development wasm binary is not available. This means the client is \
```

Comment:
Isn't the message backwards? We only set `BUILD_DUMMY_WASM_BINARY` when running clippy linter, and therefore any artifact built is not intended for production.

wasm_binary_unwrap() during chainspec construction because a wasm blob must be present. So I would expect the message to say: "WASM binary is not available. This means the client was built with `BUILD_DUMMY_WASM_BINARY` flag and is not intended for production chains. Please rebuild with the flag disabled."

Reply:
Interestingly, the wasm builder version is bumped to v2.0.0 so now I was actually able to run clippy without need of setting `BUILD_DUMMY_WASM_BINARY=1` and it seemed to work without issues.

So we might be able to drop it from our build scripts.

Reply:
It is used only during the genesis config creation. This comment states that "Development wasm binary is not available" - I understand it as "You could not use 'development wasm' in this context". Anyway, I copied it from the parity node project and could use your suggested comment. Let me know if you want it changed.

Reply:
I suppose there is not much point changing it, the key thing is the chainspec build step will fail when we need it to which is all that matters. Although in the past I have made the mistake of trying to run a node where the wasm bytes was empty, a panic occurs on execution of the node, I like that it is now caught earlier.

And the fact that we are interpreting the statement differently suggests to me that trying to open a PR upstream to fix the message is going to lack objectivity so I will spare myself :)